### PR TITLE
[FLOC-2148] Release notes floc 2148

### PR DESCRIPTION
**Pull request by adamtheturtle**
Fixes https://clusterhq.atlassian.net/browse/FLOC-2148

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -11,6 +11,9 @@ Flocker 0.4.1dev1 (2015-05-25)
 - There are new systemd units: ``flocker-zfs-agent`` is now ``flocker-container-agent`` and ``flocker-dataset-agent``. (FLOC-1554)
 - ``flocker-reportstate`` and ``flocker-changestate`` have been removed. (FLOC-1663)
 - Updated to Twisted 15.1. (FLOC-1707)
+- Dataset backend support for OpenStack and AWS. (FLOC-1925, FLOC-1743, FLOC-1745)
```
Comment (adamtheturtle) [inline NEWS]: This means that the NEWS are slightly out of sync with the title but we will do a release soon to fix this.

File: docs/releasenotes/index.rst [hunk 2]
```diff
@@ -9,14 +9,21 @@ Release Notes
 Next Release
 ============
 
-* ``flocker-deploy`` now takes an extra argument, the hostname of the control service.
+* Dataset backend support for AWS Elastic Block Storage (EBS), and OpenStack Cinder.
+* Third parties can write Flocker storage drivers so that their storage systems work with Flocker.
```
Comment (adamtheturtle) [inline docs/releasenotes/index.rst]: Ideally we'd link to a plugin here. @ferrantim said that this might come soon.

**Comment by adamtheturtle** [comment]
@ferrantim said he'd like to look at this, and that @robhaswell should too.

**Comment by ferrantim** [comment]
Got comments from @robhaswell (who is fine with it) and @lukemarsden who had a suggestion:

Luke Marsden:
"flocker-deploy now takes an extra argument, the hostname of the control service."
could be
"flocker-deploy now takes an extra first argument, the hostname of the control service."
that way users who are familiar with flocker-deploy will know to add the hostname of the control service as the *first* argument

Also, should add back: Added an API endpoint to list the nodes currently in the cluster.

Otherwise, I'm happy.

**Comment by exarkun** [comment]
> "flocker-deploy now takes an extra first argument, the hostname of the control service."

You can't have an extra first argument.  Only one argument can be first.

"flocker-deploy now requires the hostname of the control service as its first argument."  Though I think this is more detail than required in release notes.  Exact usage information is what `flocker-deploy --help` is for.

**Comment by adamtheturtle** [comment]
```
Adam Dangoor: can I merge the release notes if I make the suggested change?
Adam Dangoor: JP's
Michael Ferranti: yes
```

Therefore I will merge this when the docs builder is green.

**Pull request merged**